### PR TITLE
Development sqlalchemy

### PR DESCRIPTION
**Pull request by dpopleton**

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -2,3 +2,5 @@ mysqlclient
 pymysql
 sqlalchemy
 types-pymysql
+fastapi
```
Comment (vinay-ebi) [inline requirements.in:5]: can remove fastapi for now

File: src/ensembl/production/metadata/api.py [hunk 2]
```diff
@@ -12,26 +12,35 @@
 import sqlalchemy as db
 from sqlalchemy.orm import Session
 import pymysql
-
+import models
```
Comment (vinay-ebi) [inline src/ensembl/production/metadata/api.py:15]: test case failed as models module not found either you can use relative import  or `import  enseml.production.metadata.models`

File: src/ensembl/production/metadata/api.py [hunk 3]
```diff
@@ -12,26 +12,34 @@
 import sqlalchemy as db
```
Comment (marcoooo) [inline src/ensembl/production/metadata/api.py:12]: please add the NOTICE statemtn

**Review by marcoooo — CHANGES_REQUESTED** [review]
See my comments is the other PR #10

**Pull request merged**